src/components/network/map-equipments.js

```diff
@@ -198,7 +198,7 @@ export default class MapEquipments {
     }
 
     completeSubstationsInfos(equipementsToIndex) {
-        const nominalVoltagesSet = new Set(this.nominalVoltages);
```

Comment:
It was specifically added in this PR #1470 to fix a bug apparently, why the removal ?